config.json

```diff
@@ -274,7 +274,14 @@
       "core": false,
       "difficulty": 4,
       "slug": "collatz-conjecture",
-      "topics": null,
+      "topics": [
+        "exception_handling",
+        "recursion",
+        "conditionals",
+        "equality",
+        "integers",
+        "mathematics"
```

Comment:
Great suggestions! :) The only one I'm not sure about is `equality`. How do you think that's relevant to the exercise?

Reply:
I was thinking about the termination check `start == 1` and the check if the number is even `start % 2 == 0`
but if the equality topic is more about equality of objects, i'll remove it :-)

Reply:
Yes, I think the equality topic should be more about equality of objects, otherwise we could add it to almost every exercise :) Thanks for explaining! :)